### DISCUSSION TITLE
v8n - Validation

### DISCUSSION THREAD

**Discussion by brunotourinho** [fomantic/Fomantic-UI#952] [General] [answered]
# v8n - Validation Integration

Hey, wonderful people, working hard to make us happy developers! 😄

Today I found this library [v8n](https://github.com/imbrn/v8n)

It seems very nice, I was thinking if we can somehow integrate it with FUI. 🤔

Waiting for the experts analysis!

Thank you guys!

**Comment by prudho** [comment, marked as answer]
Well, it can be used with FUI trough validation's custom rules [(example)](https://jsfiddle.net/aknLx19t/2/).

**Comment by brunotourinho** [comment]
Nice, thanks @prudho  for the explanation!
